pkg/nsx/services/firewall.go

```diff
@@ -82,7 +94,12 @@ func (service *SecurityPolicyService) buildSecurityPolicy(obj *v1alpha1.Security
 	policyPriority := int64(obj.Spec.Priority)
 	nsxSecurityPolicy.SequenceNumber = &policyPriority
 
-	policyGroup, policyGroupPath, _ := service.buildPolicyGroup(obj)
+	policyGroup, policyGroupPath, err := service.buildPolicyGroup(obj)
+	if err != nil {
+		log.Error(err, "failed to build policy groups")
```

Comment:
groups -> group
Because we only create ONE policy-level group for one security policy CR.

Reply:
Thanks for correction.